content/en/cloud-computing.md

```diff
@@ -8,7 +8,15 @@ tags: ["infrastructure", "fundamental", ""]
 ## What it is
 
 Cloud computing offers compute resources like CPU, network, and disk capabilities on-demand over the internet, allowing users to access and use computing power in a remote physical location. 
-We generally differentiate between private and public cloud, depending on whether the cloud infrastructure is exclusively dedicated to an organization or shared for open public services.
+It extends beyond merely providing resources and we generally differentiate between private and public cloud. These models differ based on the usage and accessibility of the cloud infrastructure.
```

Comment:
Not sure I understand. What does it provide that extends beyond providing resources?

Reply:
Providing Resources in the sense, apart from providing computational needs, we have models based on our requirements. Basically we can cover specific use cases. I have quoted more on the Private, public and hybrid cloud solutions which was proposed in the issue itself.

Reply:
Hmm...I think in private, public, and hybrid clouds, you are still getting resources. The question is, who do you share those resources with? I would say, at the core, they offer virtualized resources. Cloud providers also provide PaaS (e.g., AKS), which I belive is more of an extension of their cloud computing services. 

But I may be wrong. Let's ask some of our technical team to chime in. @nate-double-u @iamNoah1 @castrojo, any thoughts?